### PR TITLE
Cascade R-CNN export

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

File: mmdet/models/detectors/cascade_rcnn.py [hunk 1]
```diff
@@ -403,6 +407,28 @@ def simple_test(self, img, img_meta, proposals=None, rescale=False):
 
         return results
 
+    def postprocess(self, det_bboxes, det_labels, det_masks, img_meta, rescale=False):
+        img_h, img_w = img_meta[0]['ori_shape'][:2]
+        scale_factor = img_meta[0]['scale_factor']
```
Comment (Ilya-Krylov) [inline mmdet/models/detectors/cascade_rcnn.py]: is it scalar?

File: mmdet/models/anchor_heads/rpn_head.py [hunk 2]
```diff
@@ -64,6 +64,7 @@ def get_bboxes_single(self,
         for idx in range(len(cls_scores)):
             rpn_cls_score = cls_scores[idx]
             rpn_bbox_pred = bbox_preds[idx]
+            anchors = mlvl_anchors[idx]
```
Comment (Ilya-Krylov) [inline mmdet/models/anchor_heads/rpn_head.py]: how did it work before?

File: tools/export.py [hunk 3]
```diff
@@ -190,21 +190,25 @@ def symbolic(self, g, featmap):
...
             anchor_generators[i].grid_anchors.params['base_anchors'] = anchor_generators[
                 i].base_anchors.cpu().numpy()
 
+def main(args):
+    model = init_detector(args.config, args.checkpoint, device="cpu")
+    model.eval().cuda()
+    torch.set_default_tensor_type(torch.FloatTensor)
+
+    if not args.no_anchor_stub:
+        stub_anchor_generator(model.rpn_head)
```
Comment (Ilya-Krylov) [inline tools/export.py]: looks like it does not work for models that do not have rpn

File: mmdet/models/detectors/cascade_rcnn.py (hunk 1 above)
Comment (druzhkov-paul) [reply mmdet/models/detectors/cascade_rcnn.py]: For all available models -- yes.

File: mmdet/models/anchor_heads/rpn_head.py (hunk 2 above)
Comment (druzhkov-paul) [reply mmdet/models/anchor_heads/rpn_head.py]: It didn't. Probably it's been broken during merge. Moved this line to better align with the current master branch state.

File: tools/export.py (hunk 3 above)
Comment (druzhkov-paul) [reply tools/export.py]: Yeah, fixed.

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**